.github/workflows/sycl_mlir_weekly.yml

```diff
@@ -0,0 +1,89 @@
+name: SYCL MLIR Weekly
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 22 * * THU'
+
+jobs:
+  ubuntu2204_build:
+    if: github.repository == 'intel/llvm'
+    uses: ./.github/workflows/sycl_linux_build.yml
+    secrets: inherit
+    with:
+      build_ref: sycl-mlir
+      build_cache_root: "/__w/"
+      build_artifact_suffix: default
+      merge_ref: ''
+      retention-days: 10
+
+  ubuntu2204_test:
+    needs: [ubuntu2204_build]
+    if: ${{ always() && !cancelled() && needs.ubuntu2204_build.outputs.build_conclusion == 'success' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Intel L0 GPU
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: ext_oneapi_level_zero:gpu
+            reset_gpu: true
+
+          - name: Intel OCL GPU
+            runner: '["Linux", "gen12"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: opencl:gpu
+            reset_gpu: true
+
+          - name: OCL CPU
+            runner: '["Linux", "x86-cpu"]'
+            image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:latest
+            image_options: -u 1001
+            target_devices: opencl:cpu
+
+    uses: ./.github/workflows/sycl_linux_run_tests.yml
+    with:
+      name: ${{ matrix.name }}
+      runner: ${{ matrix.runner }}
+      image: ${{ matrix.image }}
+      image_options: ${{ matrix.image_options }}
+      target_devices: ${{ matrix.target_devices }}
+      reset_gpu: ${{ matrix.reset_gpu }}
+      ref: ${{ github.sha }}
+      merge_ref: ''
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.ubuntu2204_build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.ubuntu2204_build.outputs.artifact_decompress_command }}
+
+  sycl-bench_mlir_build:
+    needs: [ubuntu2204_build]
+    uses: ./.github/workflows/sycl-bench_build_and_test.yml
+    with:
+      repository_owner: whitneywhtsang
```

Comment:
@stdale-intel , is it something we can accept? It seems some SYCL MLIR tests are kept in a private repo instead of our organization.

Reply:
It is a [public repository](https://github.com/whitneywhtsang/sycl-bench) forked from sycl-bench with some modifications to support SYCL-MLIR

Reply:
I agree with Andrei here, if this is going to be part of CI, should it be mirrored in a repo under org control? Just in case at some point Whitney is no longer at here.